apps/app/components/issues/EditorBubbleMenu.tsx

```diff
@@ -0,0 +1,116 @@
+import { BubbleMenu, BubbleMenuProps } from "@tiptap/react";
```

Comment:
Rename this component file name as the rest of the components with small letters. 

Create a new folder in the components folder with the name `tiptap-editor` and place everything related to the editor in that folder.

Reply:
Done 🙌